en/admin/control.rst

```diff
@@ -2142,3 +2162,358 @@ The following shows how to use the CUBRID Manager (hereafter, CM) Administrator
+
+::
+
+    demodb_java.log
+
+If any exception during performing java stored procedure/function occurs from JVM, the exception string is saved into the java log.
+
+::
+
+    SEVERE:
+    java.lang.NullPointerException
+    at Test.testFunction(Test.java:50)
+    ...
+    at com.cubrid.jsp.StoredProcedure.invoke(StoredProcedure.java:263)
+    at com.cubrid.jsp.ExecuteThread.run(ExecuteThread.java:197)
+
+.. _cubrid-javasp-server-errors:
+
+CUBRID Java SP Server Errors
+----------------------------
+
+The following are error messages about the errors which can be occurred in starting Java SP server.
+Error messages are written to **$CUBRID/log**/\ *<db_name>_java*\ **.err**.
+
++-------+----------------------------------+-----------------------------------------------------+-----------------------------------------------------------------------------------+
+| Error | Error Message                    | Description                                         | Solution                                                                          |
+| Code  |                                  |                                                     |                                                                                   |
++=======+==================================+=====================================================+===================================================================================+
+| -900  | Java VM library is not found: ?  | CUBRID can't find the JVM library                   | Make sure JAVA_HOME or JVM_PATH variable is set properly.                         |
+|       |                                  | from the JAVA_HOME or JVM_PATH variables            | see :ref:`cubrid-javasp-environment-configuration`.                               |
++-------+----------------------------------+-----------------------------------------------------+-----------------------------------------------------------------------------------+
+| -901  | Java VM can not be started: ?    | Unexpected internal error occured in JVM library.   | Try installing the JRE again. If you keep getting the error,                      |
+|       |                                  | The JVM library may be broken, or there may be a    | try installing a different version of the JRE.                                    |
+|       |                                  | problem with the $CUBRID/java/jspserver.jar file.   | Try replacing it with the same CUBRID version of $CUBRID/java/jspserver.jar file. |
++-------+----------------------------------+-----------------------------------------------------+-----------------------------------------------------------------------------------+
+
+The following are error messages about the errors which can be occrured when there is a problem with the connection to Java SP server including it is not started.
```

Comment:
The following --> The followings

Reply:
As far as I know, "The following" is used to introduce a list, a quotation, etc and it remains the same for both singular and plural.